torch/utils/_sympy/functions.py

```diff
@@ -310,3 +310,11 @@ def eval(cls, *args):
                 [int(a) for a in stride_args]
             )
         return None
+
+
+class Round(sympy.Function):
+    is_integer = True
```

Comment:
Why has the `eval` method gone? I think optimizing away round when the input is already an integer makes sense.

Reply:
Yeah, fair enough. If we are not in `RoundDecimal`, we can do that.